x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.test.ts

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loggerMock } from '@kbn/logging/target/mocks';
+import {
+  elasticsearchServiceMock,
+  savedObjectsClientMock,
+} from '../../../../../src/core/server/mocks';
+import {
```

Comment:
I would move this import after the mock imports

Reply:
The imports are sorted automatically by the typescript language server.